app/lib/cashu/utils.ts

```diff
@@ -32,6 +39,10 @@ export const getCashuUnit = (currency: Currency) => {
   return currencyToUnit[currency];
 };
 
+export const getCashuProtocolUnit = (currency: Currency) => {
```

Comment:
can you pls add some comment for this method and one above to explain why we have both. the gist is that we don't like the fact that in "cashu protocol" they use unit usd when the value is cent and we wanted to fix that to be called cent so we introduced concept of cashu unit and cashu protocol unit